lib/nexmo_developer/app/controllers/application_controller.rb

```diff
@@ -32,6 +34,25 @@ def authenticate_admin!
     redirect_to root_path unless current_user.admin?
   end
 
+  def redirect_vonage_domain
+    return unless Rails.env.production? || Rails.env.test?
+
+    check_redirect_for(request)
+  end
+
+  def check_redirect_for(request)
+    case request.host
+    when 'developer.nexmo.com'
+      redirect_to("https://developer.vonage.com#{request.fullpath}",
+                  status: :moved_permanently) and return
+
+    when 'developer.nexmocn.com'
+      # TO-DO: LOCALE change this to point to the new domain with chinese content
+      redirect_to("https://developer.vonage.com#{request.fullpath}",
```

Comment:
@marcoranieri is there a way to change locale using a GET param? Maybe investigate how the language selection dropdown works to find a way to replicate that and still keep the locale that was done based on the subdomain. 
I'm trying to prevent Chinese readers going to the English version when being redirected from that domain.